inspirehep/utils/record_getter.py

```diff
@@ -102,6 +102,10 @@ def get_es_record_by_uuid(uuid):
 
 @raise_record_getter_error_and_log
 def get_db_record(pid_type, recid):
-    from inspirehep.modules.records.api import InspireRecord
-    pid = PersistentIdentifier.get(pid_type, recid)
-    return InspireRecord.get_record(pid.object_uuid)
+    try:
+        from inspirehep.modules.records.api import InspireRecord
+        pid = PersistentIdentifier.get(pid_type, recid)
+        return InspireRecord.get_record(pid.object_uuid)
+    except:
```

Comment:
You should capture a specific error here.

And not use print but `current_app.logger.info/warn/error`